bHY6yh-d1e792/cod-pu6y7d_bHY6yh-d1e792.xml

```diff
@@ -60,43 +60,43 @@
                ¶ De munere diuino. Rubrica. 4.
             </p>
             <p xml:id="bHY6yh-d1e792-d1e119">
-               <quote xml:id="bHY6yh-d1e792-Qd1e121">
-                  <lb ed="#V" n="58"/>PSalmista. Psalmus 36.
-                  <lb ed="#V" n="59"/>Apud dominum
+               ><ref xml:id="bHY6yh-d1e792-Qd1e121">
+               <lb ed="#V" n="58"/>PSalmista. Psalmus 36. </ref>
+                  <quote <lb ed="#V" n="59"/>Apud dominum
```

Comment:
Same issue as described in the other Pull request. 

Also at line 65. The `<quote>` should start to the right of the line break rather than the left.